resources/sass/app.scss

```diff
@@ -0,0 +1,93 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Source+Serif+Pro&display=swap');
+
+body {
+    padding: 24px;
+}
+
+a, .tabs a:hover {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+header {
+    display: flex;
+    justify-content: space-between;
+
+    h1 {
+        font-family: 'Source Serif Pro', serif;
+        font-size: 32px;
+        line-height: 40px;
+        margin: 0;
+        padding: 0 0 28px 0;
+    }
+
+    .username {
+        font-family: 'Lato', sans-serif;
+        font-size: 16px;
+        line-height: 24px;
+        color: #3366cc;
+
+        a {
+            margin: 0 5px;
+        }
+    }
+    .icon-user {
+        margin-right: 8px;
+        vertical-align: middle;
+    }
+}
+
+nav.tabs {
+    display: flex;
+    padding-bottom: 25px;
+    ul {
+        list-style: none;
+        display: flex;
+        font-family: 'Lato', sans-serif;
+        font-size: 16px;
+        margin: 0;
+        padding: 0;
+        font-weight: 700;
+        border-bottom: 1px solid #A2A9B1;
+    } 
+    li {
+        padding: 4px 10px;
+        border: 2px solid white;
+    
+        &:hover a {
+            color: #447ff5;
+        }
+    
+        &:hover {
+            border-color: #447ff5;
```

Comment:
One more thing I noticed This gives the menu item a full border, is that supposed to be the case or should just be `boreder-bottom`?

![Screenshot 2021-07-23 at 10-28-46 Mismatch Finder](https://user-images.githubusercontent.com/6132917/126756645-5586edb1-7c3d-4bf7-b076-518088a9b212.png)

Reply:
Just bottom. Correcting now. :)